ethcore/src/engines/authority_round/mod.rs

```diff
@@ -806,19 +813,25 @@ impl AuthorityRound {
 			return;
 		}
 
-		if let (true, Some(me)) = (current_step > parent_step + 1, self.signer.read().as_ref().map(|s| s.address())) {
+		if let (true, me) = (current_step > parent_step + 1, self.address()) {
 			debug!(target: "engine", "Author {} built block with step gap. current step: {}, parent step: {}",
```

Comment:
This will print out the warning even for non-authoirities now, maybe let's check `me.is_zero()` to avoid printing this for regular nodes?